lib/controller/handlers/callback.js

```diff
@@ -1,3 +1,4 @@
+import isObject from 'lodash/isObject'
```

Comment:
Let's do the check without adding lodash as a dependency. It should be a simple test for us. For example test for boolean:
```
const isBoolean = (value === false || value === true)
```

Reply:
or see:
<img width="1470" alt="image" src="https://github.com/toriihq/hapi-saml2/assets/170214/c630db6d-5dce-49b4-a0f4-04f96501bfcf">
https://youmightnotneed.com/lodash

We prefer to have no dependencies at all if possible for npm modules we publish

Reply:
Removed.